agir/events/admin/panels.py

```diff
@@ -442,9 +442,12 @@ def rsvps_buttons(self, object):
 
     def participants_display(self, obj):
         if obj:
+            label = str(obj.participants)
+
             if obj.participants != obj.participants_confirmes:
-                return f"{obj.participants} participants (dont {obj.participants_confirmes} confirmes)"
-            return str(obj.participants)
+                label = f"{obj.participants} participants (dont {obj.participants_confirmes} confirmes)"
```

Comment:
Pas sûr du sens de la phrase mais c'est `confirment` ou `confirmations` non ?
```suggestion
                label = f"{obj.participants} participants (dont {obj.participants_confirmes} confirment)"
```

Reply:
Je crois que c'est `confirmés` plutôt, je vais corriger